src/style.css

```diff
@@ -2,2 +1,76 @@
+* {
   margin: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url('./fonts/Roboto-Medium.ttf') format('truetype');
+  font-weight: 500;
+  font-style: normal;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  vertical-align: middle;
+}
+
+.menu__list {
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+.menu__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
+}
+
+.menu__list .menu__item:nth-child(4) .menu__link:hover {
+  color: rgb(0, 172, 220);
```

Comment:
The hover effect is applied only to the fourth menu item. If this is intentional, ensure that it aligns with the design requirements. Otherwise, consider applying hover effects consistently across all menu items.